tests/Doctrine/Tests/Common/Cache/PhpFileCacheTest.php

```diff
@@ -66,6 +66,17 @@ protected function _getCacheDriver()
     {
         return new PhpFileCache($this->directory);
     }
+
+    public function testFailureOnIncludeNotSettedCacheDoesntGetCaughtAsError()
+    {
+        $cache = $this->_getCacheDriver();
+        set_error_handler(function($errno, $errstr, $errfile, $errlin){
+            $this->fail('include failure captured');
+        });
+        $cache->fetch('key');
```

Comment:
Maybe test for ``$cache->delete('not_existing_key');`` could also be added - it has the same problem